hudi-common/src/main/java/org/apache/hudi/common/model/DeleteRecord.java

```diff
@@ -88,6 +88,13 @@ public Comparable<?> getOrderingValue() {
     return orderingVal;
   }
 
+  public Comparable<?> getOrderingVal(Comparable<?> defaultVal) {
+    if (orderingVal == null) {
```

Comment:
replace orderingVal with getOrderingValue()

Reply:
? It is not as efficient as check orderingValue directly, right? For extensibility?

Reply:
or for back compatibility?

Reply:
I guess if someone overrides getOrderingValue(). But it's probably fine to leave it as is